packages/plugin-rem/src/types.ts

```diff
@@ -3,13 +3,53 @@
  * https://github.com/cuth/postcss-pxtorem#options
  */
 export type PxToRemOptions = {
+  /**
+   * (Number | Function) Represents the root element font size or returns the root element font size based on the input parameter
```

Comment:
TypeScript already provides type definitions, so we may not need to declare the type in the JSDoc